exercises/atbash-cipher/example.go

```diff
@@ -12,10 +12,8 @@ func Atbash(s string) string {
 }
 
 func chunk(s string) string {
-	reg, _ := regexp.Compile(".{1,5}")
-	value := reg.FindAllString(s, -1)
-	s = strings.Join(value, " ")
-	return s
+	value := regexp.MustCompile(".{1,5}").FindAllString(s, -1)
```

Comment:
Maybe in another PR:
`regexp.MustCompile` should be executed only once per regex pattern. Since it's result is basically static I don't see a problem using a global for it.